firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -839,6 +839,8 @@ class BrowserViewController: UIViewController,
         if isToolbarRefactorEnabled {
             addressToolbarContainer.configure(windowUUID: windowUUID, profile: profile, delegate: self)
             addressToolbarContainer.applyTheme(theme: currentTheme())
+            let isPrivate = tabManager.selectedTab?.isPrivate ?? false
+            addressToolbarContainer.applyUIMode(isPrivate: isPrivate, theme: currentTheme())
```

Comment:
is there a way to avoid `applyUIMode` for the new code? It's something we eventually would like to remove eventually and would like to avoid adding more code using it if possible. 
Ticket: https://mozilla-hub.atlassian.net/browse/FXIOS-8907